x-pack/legacy/plugins/siem/server/lib/detection_engine/signals_mapping.json

```diff
@@ -0,0 +1,1644 @@
+{
```

Comment:
@dhurley14 , check these fields against:
https://github.com/elastic/ecs/blob/v1.1.0/generated/csv/fields.csv

To see if I missed any (it is very likely as I did this very quickly).

Reply:
These are the fields I found missing from the signals_mappings.json

https://gist.github.com/dhurley14/ff54a356d0ab07ddbd10b6c48b980d2d